builq.go

```diff
@@ -78,28 +61,40 @@ func (b *Builder) writeArgs(s fmt.State, verb rune, arg any, isMulti bool) {
 			fmt.Fprint(s, ", ")
 		}
 
-		b.appendArg(arg, verb)
-
 		switch verb {
 		case '$': // PostgreSQL
+			b.counter++
```

Comment:
Looks like a better place for this line, 'cause the counter is used only here, WDYT?

Reply:
After `appendArg` merge it's ok, thanks.